core-lightning-rtl/umbrel-app.yml

```diff
@@ -44,10 +44,7 @@ gallery:
 path: ""
 deterministicPassword: true
 releaseNotes: >-
-  ⚠️ Please update your Core Lightning app to the latest version available in the umbrelOS app store to ensure compatibility with this version of Ride The Lightning.
-
-  
-  This update adds Boltz as a service, allowing users to swap in and out of the Lightning Network.
+  This updates Boltz Client to version 2.7.1. Release notes for Boltz Client can be found at https://github.com/BoltzExchange/boltz-client/releases.
```

Comment:
I would suggest to add a small update summary before the link to detailed release notes.

Reply:
Adressed.